src/main/java/lotto/domain/Draw.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Draw {
+
+    private static final int UNIT_PRICE = 1000;
+
+    private final int numberOfLotto;
+
+    private final List<Lotto> lottos;
+
+    private Winnings winnings;
+
+    public Draw(int totalPrice) {
+        this.numberOfLotto = totalPrice / UNIT_PRICE;
+        this.lottos = new ArrayList<>();
+        this.winnings = null;
+    }
+
+    public void drawLottos() {
+        for (int i = 0; i < numberOfLotto; i++) {
+            this.lottos.add(new Lotto());
+        }
+    }
+
+    public void checkWinnings(Winnings winnings) {
+        this.winnings = winnings;
+        for (Lotto lotto : this.lottos) {
+            lotto.checkWinnings(winnings);
+        }
+    }
+
+    public List<Lotto> lottos() {
+        return Collections.unmodifiableList(lottos);
+    }
```

Comment:
```suggestion
    public Lottos lottos() {
        return Lottos(lottos);
    }
```

컬렉션을 불변으로 만들기 위해서 `Collections.unmodifiableList` 를 활용하신 점 좋습니다. 👍 
기왕 컬렉션을 활용하실거라면, Lottos 라는 일급 컬렉션을 클래스로 선언해보시는 것은 어떠실지 제안을 해봅니다.

일급 컬렉션을 만드실 때, 불변객체가 될수 있도록 지금처럼 `Collections.unmodifiableList` 을 활용해주시면 좋을 것 같습니다.